vuepress/docs/next/docs/getting-started/ent-api.md

```diff
@@ -61,17 +61,18 @@ Common operations associated with API claims are detailed below. To execute `ent
 
 ## Microservice URL Retrieval
 
-To render an MFE installed from a bundle, Entando injects a JSON object containing configuration details inside the "config" attribute of a custom HTML element, e.g. `<my-mfe config="{ ... }" />`.
+To retrieve the URL of an MS declared through an API claim, add `systemParams.api[claimName].url` to the MFE JavaScript code so it can access that element of the config object. The `claimName` is chosen by the user when the API claim was defined.
+
+Refer to this [Entando demo bundle](https://github.com/entando/entando-showcase-bundle) for an alternative [API URL retrieval method](https://github.com/entando/entando-showcase-bundle/blob/master/microfrontends/another-mfe/src/AnotherMfe.js#L14) which considers the possibility that an element of `systemParams.api[claimName]` is undefined.
```

Comment:
I'm uneasy about linking like this, especially deeply to a specific line of code. That seems like a guarantee for the link to break over time.

Reply:
@zonia3000 could you please clarify what that code is supposed to represent? Typically we avoid linking to URLs that are outside of a regular content/docs process. (One exception is CRDs within entando-releases where we need to show code templates.) Can we just use a code fragment here instead?

Reply:
@nshaw The reference to the code could be replaced by a sentence that reminds the user to check if the parent object is undefined. There are many ways to do it, like in the linked code or e.g.:

```
if (systemParams && systemParams.api && systemParams.api[claimName]) {
    // access systemParams.api[claimName].url here
}
```
The main point is that the user can retrieve the API URL from `systemParams.api[claimName].url`, but he/she has to be aware that in some conditions calling that code directly, without any check, could result in an error.

Probably is not so important, it's a detail. We could also remove this remind if you think that it creates confusion.

Reply:
@zonia3000 thanks, I would remove it then. That looks like an implementation detail and a fairly common FE practice anyway.